groups/chips-uam-infrastructure/templates/chips_uam_user_data.tpl

```diff
@@ -25,14 +25,15 @@ rm -rf /home/ec2-user/webswing-2.5.5-distribution.zip
 
 #Copy master.txt from vault to /home/ec2-user/uam
 #Create key:value variable
-cat <<EOF >> /home/ec2-user/uam/master.txt
+cat <<EOF >> /home/ec2-user/uam_gui-1.109.0-rc1/master.txt
 ${CHIPS_UAM_INPUTS}
 EOF
 
-chmod 700 /home/ec2-user/uam/master.txt
+chmod 700 /home/ec2-user/uam_gui-1.109.0-rc1/master.txt
 
 #Encrypt master.txt
-/home/ec2-user/uam/encryptUamDbConfigFile.sh master.txt
+cd /home/ec2-user/uam_gui-1.109.0-rc1/
+/home/ec2-user/uam_gui-1.109.0-rc1/encryptUamDbConfigFile.sh master.txt
 
```

Comment:
Add in a remove command to remove the unencrypted master.txt file.

Reply:
command to remove unencrypted master.txt:

rm -rf /home/ec2-user/uam_gui-1.109.0-rc1/master.txt